app/dashboard/migrations/0056_auto_20180423_1343.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.0.4 on 2018-04-23 13:43
```

Comment:
You'll need to rebase with `master`, delete this file, and recreate your migrations via: `make migrations`